### PR TITLE
Create Card Methods

### LINKED ISSUE #5: Implement Card resource methods
## Overview

Now that Board and List methods are implemented, we need to be able to interact with Cards on Lists.

## Tasks 

- [x] `get_card()`
- [x] `create_card()`
- [x] `update_card()`
- [x] `delete_card()`
  
## Acceptance Criteria

- Write tests for Board methods
- Document Board methods as doc strings

### PR DESCRIPTION
**Pull request by purplemoon-l**
Included all 4 functionalities of Card
Removed Linter and formatting details
Fixes #5

### CODE REVIEW

File: pytrello2/client.py [hunk 1]
```diff
@@ -44,11 +44,12 @@ def get(self, endpoint, params=None):
...
         return self._request("GET", endpoint, params=params)
 
-    def post(self, endpoint, data=None):
+    def post(self, endpoint, params=None):
         """
         Sends a POST request to the Trello API.
         """
-        return self._request("POST", endpoint, data=data)
+
+        return self._request("POST", endpoint, params=params)
```
Comment (vchrombie) [inline pytrello2/client.py:52]: Is this a required change?

File: .gitignore [hunk 2]
```diff
@@ -6,6 +6,9 @@ __pycache__/
 # C extensions
 *.so
 
+*.DS_Store
+.DS_Store
+
```
Comment (vchrombie) [inline .gitignore:11]: This can be removed

**Review by vchrombie — CHANGES_REQUESTED** [review]
Good work on the DCO, can you these comments?

**Comment by vchrombie** [comment]
Please update the changes in the PR and description

File: pytrello2/client.py (hunk 1 above)
Comment (purplemoon-l) [reply pytrello2/client.py]: Fixed it

File: .gitignore (hunk 2 above)
Comment (purplemoon-l) [reply .gitignore]: Removed

File: tests/test_models.py [hunk 3]
```diff
@@ -28,7 +30,18 @@ def test_board_model():
     assert board.url == mock_data["url"]
 
 
-# Test for board str method
```
Comment (vchrombie) [inline tests/test_models.py:31]: Can you revert this?

Comment (purplemoon-l) [reply tests/test_models.py]: Yes Done

**Review by vchrombie — APPROVED** [review]
LGTM! 🚀

**Pull request merged**